### ISSUE TITLE
Conda issue during numpy 1.11 update

### ISSUE THREAD

**Issue by MSeifert04** [astropy/astroscrappy#15]
Is there any reason the conda build doesn't allow numpy 1.11? I get this error when trying to update:

```
[Anaconda] C:\Users\Admin>conda install numpy=1.11
Using Anaconda Cloud api site https://api.anaconda.org
Fetching package metadata: ......
Solving package specifications: ....

The following specifications were found to be in conflict:
  - astroscrappy (target=astroscrappy-1.0.3-np110py35_0.tar.bz2) -> numpy 1.10*|1.9*
  - astroscrappy (target=astroscrappy-1.0.3-np110py35_0.tar.bz2) -> python 2.7*|3.4*
  - numpy 1.11*

Use "conda info <package>" to see the dependencies for each package.
```

**Comment by astrofrog** [comment]
It's possible that the conda package with Numpy 11 hasn't been built yet (@mwcraig, is that correct?)

**Comment by mwcraig** [comment]
@MSeifert04 @astrofrog -- looks like continuum has now released astropy builds for numpy 1.11 (it typically takes them about a month to build out all of the numpy dependent packages after a new numpy release is made).

I'll make getting the affiiated packages built a priority for Thursday morning. In the meantime you can use the `-f` option to force the numpy upgrade. Newer versions of numpy try hard to be backwards-compatible in their ABI, so an astroscrappy built against numpy 1.10 should work with numpy 1.11.

**Comment by MSeifert04** [comment]
@mwcraig Thank you!

**Comment by mwcraig** [comment]
@MSeifert04 -- astroscrappy is now available for numpy 1.11 on either of the conda channels `astropy` or `openastronomy`

**Comment by MSeifert04** [comment]
Finally I'm able to update numpy. Thank you!
